templates/default/ndb-agent/config.yml.erb

```diff
@@ -12,6 +12,8 @@ http:
 heartbeat:
   enabled: HEARTBEAT_ENABLED
   endpoint: HEARTBEAT_ENDPOINT
+  is-leader: HEARTBEAT_ENABLED
+  cloud-base-endpoint: HEARTBEAT_ENDPOINT
```

Comment:
I'm not sure why there was a need for changing `endpoint` to `cloud-based-endpoint` or what cloud-based is supposed to mean but anyway

Reply:
Agree :) but let's go with it.